dashboard/app/controllers/courses_controller.rb

```diff
@@ -20,4 +29,21 @@ def show
 
     render 'show', locals: {course: course}
   end
+
+  def new
+    render 'new'
+  end
+
+  def create
+    course = Course.new(name: params.require(:course).require(:name))
```

Comment:
They use two different sets of params? `show` takes `params[:course_name]`, where as here we have `params[:course][:name]`